Src/FluentAssertions/Xml/XmlReaderValidator.cs

```diff
@@ -12,9 +13,32 @@ internal class XmlReaderValidator
         private readonly XmlReader subjectReader;
         private readonly XmlReader otherReader;
 
-        private string GetCurrentLocation() => "/" + string.Join("/", locationStack.Reverse());
+        private string GetCurrentLocation()
```

Comment:
I recommend renaming it to `GetCurrentXPath`. We also prefer to order methods by invocation so you can read from top to bottom. In this case, it means moving this method under `ValidateText`.

Reply:
I will do so as soon you have merged #1174.

Reply:
Done